src/sst/core/Makefile.am

```diff
@@ -116,6 +116,7 @@ nobase_dist_sst_HEADERS = \
 	interprocess/circularBuffer.h \
 	interprocess/sstmutex.h \
 	interprocess/ipctunnel.h \
+	nlohmann/json.hpp \
```

Comment:
I think we should have a ```contrib/nlohmann` directory - a place where we can put any contributed source code (seeing as we may have others we add over time)?

Reply:
Agreed.  Or "tpl" or some such.  We should also move tinyxml into that directory.

Reply:
@feldergast @nmhamster I just followed along with how the tinyxml directory was done.  Some projects do this in a top-level `/external` directory for things that aren't part of the project. This would also make it easy to not run the format tester on the files.

Reply:
Understood regarding `tinyxml`, we just need to get both of these things out into a `contrib` somewhere but I'm not wedded strongly to specifically where that goes as long as we do it.